_data/simple-icons.json

```diff
@@ -7650,7 +7654,7 @@
         {
             "title": "ResearchGate",
             "hex": "00CCBB",
-            "source": "https://c5.rgstatic.net/m/428059296771819/images/favicon/favicon.svg"
+            "source": "http://biblioteca.uoc.edu/es/recursos/recurso/researchgate"
```

Comment:
I have a 404 with this one... 😢

Reply:
Still, it does not represent the source used when adding the icon to the library since webp is a raster image format.

Reply:
The source should probably just be https://www.researchgate.net/, as the favicon that is the current source is used there. 

@jorgeamadosoria, to make it perhaps a bit easier to understand the source choice, could you include an explanation in the commit making this change? (or do it in a dedicated PR for visibility)

Reply:
Where in   https://www.researchgate.net/ is the icon? I can't find it.

Reply:
changed.